metadata-ingestion/src/datahub/sql_parsing/sql_parsing_aggregator.py

```diff
@@ -185,6 +185,10 @@ class TableSwap:
     urn1: UrnStr
     urn2: UrnStr
 
+    def id(self) -> str:
+        # TableSwap(A,B) is same as TableSwap(B,A)
+        return str(hash(frozenset([self.urn1, self.urn2])))
```

Comment:
we can do `dataclasses.dataclass(frozen=True)` to make it hashable

Reply:
I did try that earlier but it does not ignore the order of urns. Having array instead of individual fields would work but then we would need to ensure somehow that array has only two entries. Also might make code less readable, so keeping this as is.